overlay_scenes/item-manupulators/traderOverlayScene.ts

```diff
@@ -101,6 +145,8 @@ export class TraderOverlayScene extends ContainerOverlayScene {
         } else {
             this._moveItemFromSlotToFirstPossible(itemCurrentSlot, backpackSlotNames, 1)
         }
+        this.setCurrentCopperAmount()
+        this.updatePriceTags()
```

Comment:
No, I don't think we need it here :) The reason why I put it here is because I had an issue with the price tags. Sometimes after buying an item by clicking on it, some price tags didn't change to the correct color. I thought one reason could be due to a delay in the call to moveItemFromSlotToSlot, but I think it was due to an incorrect condition in the setPriceTag method.